nixos/tests/seatd.nix

```
@@ -0,0 +1,53 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+let
+  seatd-test = pkgs.writeShellApplication {
+    name = "seatd-client-pid";
+    text = ''
+      systemctl status seatd --no-pager | while read -r line; do
+          case "$line" in
+          *"New client connected"*)
+              line="''${line##*pid: }"
+              pid="''${line%%,*}"
+              ;;
+          *"Opened client"*)
+              echo "$pid"
+              exit
+          esac
+      done;
+    '';
+  };
+in
+{
+  name = "seatd";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ sinanmohd ];
+  };
```

Could probably write this a bit easier, if you import `lib` as well as pkgs on line 1

```suggestion
  meta.maintainers = with lib.maintainers; [ sinanmohd ];
```